README.rst

```diff
@@ -118,20 +118,26 @@ You can restore a sample dataset, then download the data sample and insert it in
         # or directly with curl
         curl -L "https://github.com/qwat/qwat-data-model/releases/download/$VERSION/qwat_v"$VERSION"_data_only_sample.backup" | pg_restore -U postgres --dbname qwat -e --no-owner --verbose --disable-triggers --port 5432
 
-.. Doesn't work with 1.3.1 see https://github.com/qwat/qwat-data-model/issues/224
-.. Instead of running the init script or git, just run :
+Instead of running the init script or git, just run :
```

Comment:
i don't understand the "or git" personally… I know it was like this before, but I don't find this very clear.

That being said, in my opinion we should mention at the very beginning of the Install section that there are two ways to set up the qWat database. The first way is to create a empty database using the `init_qwat.sh` script. The second way is to restore the sample dump using `pg_restore`.

On another note, I don't find it useful to provide both `wget` and `curl` command lines. I think we could just go with `wget`, which is more explicit and common to people.

Reply:
@elemoine You're right. I correct that.

Reply:
Hi, qWat is now supposed to be written "QWAT" If I get well. I think I saw some occurences in the readthedoc (title a last)